conf/nginx/server-base.conf.jnj

```diff
@@ -18,6 +18,11 @@ proxy_redirect off;
 
 proxy_set_header Transfer-Encoding $http_transfer_encoding;
 
+# Temporarily force signin for old and new UI to route to web app
+location /signin {
+    proxy_pass   http://web_app_server/;
```

Comment:
Adding a trailing `/` is required at the moment, we have a redirect to `/signin` somewhere and it ultimately leads to a bad redirect where the port is dropped.

Reply:
does nginx treat `/signin` and `/signin/` the same?

Reply:
It does. Here's my local test. I add a custom header of `X-Signin` if it hits the `/signin` location. 

![image](https://user-images.githubusercontent.com/1656866/187937457-0c83d716-ff1c-4788-8358-d33525f26131.png)
![image](https://user-images.githubusercontent.com/1656866/187937529-93476e2c-b2d7-4843-b29a-2cb4a69d3b2b.png)

Reply:
nice looks good 👍